### PR TITLE
Update structure test yaml to schema 2.0.0.

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: container/common/java.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 commandTests:
-- name: 'javahome-envvar'
-  command: ['sh', '-c', 'echo $JAVA_HOME']
-  expectedOutput: ['/usr/lib/jvm/java-8-openjdk-amd64']
 - name: 'java-version'
-  command: ['bash', '-c', 'java -version 2>&1']
-  expectedOutput: ['openjdk version \"1.8.*']
+  command: 'java'
+  args: ['-version']
+  expectedError: ["openjdk version \"1.8.*"]
```
Comment (nlopezgi) [inline container/common/java.yaml]: why is this expectedError and not expectedOutput?

File: container/common/python2.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-schemaVersion: "1.0.0"
+schemaVersion: "2.0.0"
 
 commandTests:
 - name: 'python2-version'
-  command: ['bash', '-c', 'python -V 2>&1']
-  expectedOutput: ['Python 2.7.*']
+  command: 'python'
+  args: ['-V']
+  expectedError: ['Python 2.7.*']
```
Comment (nlopezgi) [inline container/common/python2.yaml]: same here why expectedError?

File: container/common/java.yaml (hunk 1 above)
Comment (xingao267) [reply container/common/java.yaml]: Because it seems java command just output the content to stderror

File: container/common/python2.yaml (hunk 2 above)
Comment (xingao267) [reply container/common/python2.yaml]: same

File: container/common/java.yaml (hunk 1 above)
Comment (nlopezgi) [reply container/common/java.yaml]: leave comment about this being the case

Comment (xingao267) [reply container/common/java.yaml]: done

**Pull request merged**